cinema/tests/test_cinema_hall_api.py

```diff
@@ -116,9 +116,13 @@
         self.assertEqual(CinemaHall.objects.get(id=1).name, "Green")
 
     def test_delete_cinema_hall(self):
+        print("Before DELETE request:", CinemaHall.objects.all())
         response = self.client.delete(
             "/api/cinema/cinema_halls/1/",
         )
+        print("After DELETE request:", CinemaHall.objects.all())  # Друкуємо всі об'єкти
+        print("Count of CinemaHall with id=1:",
+              CinemaHall.objects.filter(id=1).count())  # Друкуємо кількість об'єктів з id=1
```

Comment:
Unnecessary print statements. Consider removing them to keep the test output clean.